app/actions/todos.go

```diff
@@ -20,3 +22,30 @@ func Index(c buffalo.Context) error {
 
 	return c.Render(http.StatusOK, r.HTML("/todos/index.plush.html"))
 }
+
+func NewTask(c buffalo.Context) error {
+	todo := models.Todo{}
+	c.Set("todo", todo)
+
+	return c.Render(http.StatusOK, r.HTML("todos/new.plush.html"))
+}
+
+func PostNewTask(c buffalo.Context) error {
+	u, _ := uuid.NewV4()
+	todo := models.Todo{ID: u}
+
+	if err := c.Bind(&todo); err != nil {
+		return err
```

Comment:
We don't return naked errors. That is because they don't provide any context when they occur. Let's use this one instead:
`c.Error(http.StatusInternalServerError, errors.Wrap(err, "Save - error while binding to Todo"))`
In here we provide the exact http code, along with contextual information for the error